includes/dbFacile.php

```diff
@@ -35,6 +35,10 @@ function dbQuery($sql, $parameters = array()) {
 	*/
 
 	$result = mysql_query($fullSql); // sets $this->result
+       	if (!empty(mysql_error())) {
```

Comment:
This needs to be:

if (mysql_error()) {

Otherwise you get:

Fatal error: Can't use function return value in write context in includes/dbFacile.php on line 38